packages/tools/fluid-runner/src/parseBundleAndExportFile.ts

```diff
@@ -58,22 +58,25 @@ export async function parseBundleAndExportFile(
 					return { success: false, eventName, errorMessage };
 				}
 
-				const argsValidationError = getArgsValidationError(inputFile, outputFile);
+				const argsValidationError = getArgsValidationError(inputFile, outputFile, timeout);
 				if (argsValidationError) {
 					const eventName = clientArgsValidationError;
 					logger.sendErrorEvent({ eventName, message: argsValidationError });
 					return { success: false, eventName, errorMessage: argsValidationError };
 				}
 
-				fs.writeFileSync(
-					outputFile,
-					await createContainerAndExecute(
+				const executeResult = await timeoutPromise<string>((resolve, reject) => {
```

Comment:
should some of the logic here move into createContainerAndExecute() or be extracted into its own function to improve code reuse with exportFile() that has similar changes?